src/styles/index.scss

```diff
@@ -1,3 +1,243 @@
+}
+
+.nav__link:hover {
+  color: #00bfff;
+}
+
+.is-active {
+  color: #00bfff;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: -19;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00bfff;
+  border-radius: 8px;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+.card {
+  border-radius: 5px;
+  width: 200px;
+  margin: 0;
+  padding: 32px 16px 16px;
+  font-family: $font-family;
+  border: 1px solid #f3f3f3;
+  transition:
+    transform 0.3s ease,
+    color 0.3s ease;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: $font-color;
+  margin-bottom: 4px;
+  transition: color 0.3s ease;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $light-grey;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  align-items: center;
+  justify-content: space-between;
+  display: flex;
+  margin-bottom: 24px;
+}
+
+.card__review_count {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: $font-color;
+}
+
+.stars {
+  display: flex;
+  gap: 4px;
+  padding-left: 1px;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star-active {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price_text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: $light-grey;
+}
+
+.card__price_count {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: $font-color;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  background-color: $primary-color;
+  color: white;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-decoration: none;
+  border-radius: 5px;
+  transition: background-color 0.3s ease;
+  cursor: pointer;
+  border: 1px solid $primary-color;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: $primary-color;
+}
+
+.card:hover .card__title {
+  color: #34568b;
+}
+
+.card:hover {
+  transform: scale(1.2);
```

Comment:
The scale transformation on hover might be too large (1.2), which could affect the layout or cause overlap with other elements. Consider using a smaller scale value to ensure a smoother user experience.